cfpb/consumerfinance.gov
# Main TDP survey grade switch page moved to /assess/survey/

The grade-selection page has a different URL than we expected. This makes the necessary updates.


---

## How to test this PR

1. Load https://beta.consumerfinance.gov/consumer-tools/educator-tools/youth-financial-education/survey/3-5/
2. The link "Switch grades" should point to: https://beta.consumerfinance.gov/consumer-tools/educator-tools/youth-financial-education/assess/survey/

  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
